visionipc/visionbuf_ion.cc

```diff
@@ -117,6 +117,27 @@ void VisionBuf::init_cl(cl_device_id device_id, cl_context ctx) {
   assert(err == 0);
 }
 
+void VisionBuf::init_gl() {
```

Comment:
any reason to do this here instead of UI?

Reply:
the image/texture is linked to the shared buffers, so it requires a texture pointer for every visionbuf object, so I guess it makes more sense to put it there